FishjamClient.podspec

```diff
@@ -0,0 +1,33 @@
+#
+# Be sure to run `pod lib lint FishjamClient.podspec' to ensure this is a
+# valid spec before submitting.
+
+
+Pod::Spec.new do |s|
+  s.name             = 'FishjamClient'
+  s.version          = '0.3.0'
```

Comment:
Lets have fresh start and change version to 0.1.0 or 1.0.0 if namespaces are changed and e2e tests from react-native repo pass.

Reply:
Idk, this will make no real difference here. And repo already have 0.2.0 released. So I would have to remove old releases.
As for 1.0.0  - this SDK is not ready to have v1 released

Reply:
That release will not work because url and package name changed.  It is strange for first package release to be versioned as 0.3.0 instead of 0.1.0 or 0.0.1. Release 0.2.0 should be removed because we do not want old releases here.

Reply:
> That release will not work because url and package name changed
I don't understand - what will not work in this release? It is already working.

> Release 0.2.0 should be removed because we do not want old releases here
What is the problem with old release? It is kind of historical release, and I don't see any issue with keeping it here. Between 0.2 and 0.3 we are just adding some new functionality into lib, it is not complete rewrite.

Reply:
ok, let it stay